js/map_utils.js

```diff
@@ -33,7 +33,9 @@ var stateStyle = {
 
 function addCentroids(map, countyCentroids) {
   
-  var tempProjection = d3.geoAlbersUsa();
+  var tempProjection = albersUsaTerritories()
```

Comment:
can we use just one projection now? there's `projection` in build_map.js. i only created `tempProjection` so i could get some dots in there for the prototype without completely breaking the vizzy. ideally we'd just assign `albersUsaTerritories` to `projection`, delete `tempProjection`, and use `projection` everywhere

Reply:
If you know how to get that done smoothly, I'm all for one projection.

Reply:
i think it's just doing what i wrote above: delete the definition of tempProjection, define projection = albersUsaTerritories, and replace any uses of tempProjection with projection.